app/person/tests.py

```diff
@@ -32,3 +39,23 @@ def test_name_label(self):
     person = Person.objects.get(id=1)
     label = person._meta.get_field('name').verbose_name
     self.assertEqual(label, 'Full name')
+  
+  def test_update_screens_button_renders(self):
+    User.objects.create_superuser(test_username, test_email, test_password)
+
+    c = Client()
```

Comment:
The client assignment could be pulled up outside of the classes, alongside the other variable assignments.

Reply:
Also updated in b490133.